crates/stackable-operator/src/commons/affinity.rs

```diff
@@ -42,8 +42,18 @@ pub struct StackableAffinity {
     pub node_selector: Option<StackableNodeSelector>,
 }
 
+/// We can not simply use [`BTreeMap<String, String>`] in [`StackableAffinity`], as the fields need to be [`Atomic`].
+/// We can not mark it as [`Atomic`], as [`crate::config::fragment::FromFragment`] is already implemented for
+/// [`BTreeMap<String, String>`].
+///
+/// We `#[serde(flatten)]` the contained [`BTreeMap<String, String>`], so `serde_yaml` can deserialize everything as
+/// expected.
```

Comment:
This probably shouldn't be a doccomment, since we don't want it in the CRD docs.

Reply:
Marking as resolved